packages/node/src/notifier.js

```diff
@@ -45,6 +45,11 @@ module.exports = (opts, userPlugins = []) => {
 
   bugsnag._logger.debug(`Loaded!`)
 
+  bugsnag.leaveBreadcrumb = function () {
```

Comment:
I figured this would be the simplest approach, monkey patching the client method to log a warning.